src/main/java/org/killbill/billing/plugin/notification/setup/EmailNotificationListener.java

```diff
@@ -260,6 +264,27 @@ public String apply(AccountEmail input) {
         emailSender.sendPlainTextEmail(ImmutableList.of(account.getEmail()), ImmutableList.copyOf(cc), emailContent.getSubject(), emailContent.getBody());
     }
 
+    private boolean muteEmailForObject(final UUID objectId, final ObjectType objectType, final TenantContext context) {
+
+        try {
+            TagDefinition muteEmailTagDefinition = osgiKillbillAPI.getTagUserApi().getTagDefinitionForName("mute_email", context);
+            if (muteEmailTagDefinition == null) {
+                return false;
+            }
+
+            List<Tag> tags = osgiKillbillAPI.getTagUserApi().getTagsForObject(objectId, objectType, false, context);
+
+            for (Tag tag : tags) {
+                if (muteEmailTagDefinition.getId().equals(tag.getTagDefinitionId())) {
+                    return true;
```

Comment:
from what i remember of our last plugin work, short circuit iteration of the `tags` list may result in leaked db handles